doc/stdlib/collections.rst

```diff
@@ -1691,74 +998,50 @@ Set-Like Collections
 .. function:: IntSet([itr])
 
    .. Docstring generated from Julia source
-   ::
 
-              IntSet([itr])
+   ..  IntSet([itr])
 
    Construct a sorted set of the integers generated by the given iterable object, or an empty set. Implemented as a bit string, and therefore designed for dense integer sets. Only non-negative integers can be stored. If the set will be sparse (for example holding a single very large integer), use :obj:`Set` instead.
 
 .. function:: union(s1,s2...)
 
    .. Docstring generated from Julia source
-   .. code-block:: julia
-
-       union(s1,s2...)
-       ∪(s1,s2)
 
    Construct the union of two or more sets. Maintains order with arrays.
 
 .. function:: union!(s, iterable)
 
    .. Docstring generated from Julia source
-   .. code-block:: julia
-
-       union!(s, iterable)
 
    Union each element of ``iterable`` into set ``s`` in-place.
 
 .. function:: intersect(s1,s2...)
 
    .. Docstring generated from Julia source
-   .. code-block:: julia
-
-       intersect(s1,s2...)
-       ∩(s1,s2)
 
    Construct the intersection of two or more sets. Maintains order and multiplicity of the first argument for arrays and ranges.
 
 .. function:: setdiff(s1,s2)
 
    .. Docstring generated from Julia source
-   .. code-block:: julia
-
-       setdiff(s1,s2)
 
    Construct the set of elements in ``s1`` but not ``s2``\ . Maintains order with arrays. Note that both arguments must be collections, and both will be iterated over. In particular, ``setdiff(set,element)`` where ``element`` is a potential member of ``set``\ , will not work in general.
 
 .. function:: setdiff!(s, iterable)
 
    .. Docstring generated from Julia source
-   .. code-block:: julia
-
-       setdiff!(s, iterable)
 
    Remove each element of ``iterable`` from set ``s`` in-place.
 
 .. function:: symdiff(s1,s2...)
 
    .. Docstring generated from Julia source
-   .. code-block:: julia
-
-       symdiff(s1,s2...)
 
    Construct the symmetric difference of elements in the passed in sets or arrays. Maintains order with arrays.
 
 .. function:: symdiff!(s, n)
```

Comment:
Something is messed up with the three signatures and the code blocks for `symdiff!`